site/layouts/shortcodes/orange-footer.html

```diff
@@ -69,7 +69,7 @@ <h3 class="accordion-header footer-heading" id="headingOne{{ $id }}">
             </button>
             <span class="d-none d-md-flex">Category</span>
           </h3>
-          <div id="collapseOne{{ $id }}" class="container-xxl accordion-collapse collapse" data-bs-parent="#accordion{{ $id }}">
+          <div id="collapseOne{{ $id }}" class="container-xxl accordion-collapse collapse" aria-labelledby="headingOne{{ $id }}" data-bs-parent="#accordion{{ $id }}">
```

Comment:
No need to add a note in the migraiton guide for all those modifications regarding the footer?